pouchdb-core/pouchdb-core.d.ts

```diff
@@ -13,9 +16,9 @@ declare namespace PouchDB {
         type AnyCallback = Callback<any, any>;
         type DocumentId = string;
         type DocumentKey = string;
+        type AttachmentId = string;
         type RevisionId = string;
         type Availability = 'available' | 'compacted' | 'not compacted' | 'missing';
-        type Attachment = string | ArrayBuffer;
```

Comment:
ArrayBuffer allows Blob and Buffer? I'm not really used to work with these.
I made type as string | Blob | Buffer. This way I CANT assing Blob to this type.